variables.tf

```diff
@@ -194,6 +194,12 @@ variable "public_access_master_instance" {
   default     = false
 }
 
+variable "allocated_ip_range" {
+  description = "Allow users to select a specific IP range for their private instances"
```

Comment:
What would be the recommended IP width for this? /16? /20? /24? /32. We should recommend something here like we have recommended for `ip_ranges` variable in the [vpc_network](https://registry.terraform.io/modules/airasia/vpc_network/google/latest?tab=inputs) module.

Reply:
Added the cidr range information